clusterloader2/testing/density/config.yaml

```diff
@@ -115,7 +115,6 @@ steps:
     Method: SchedulingThroughput
     Params:
       action: start
-      labelSelector: group = saturation
```

Comment:
Going back to my first comment:
I actually vote for making it a new measurement, and simply:
- start it at the beginning of the test
- finish it at the end of the test
- not touch the existing measurement

Once we prove it's stable, we will simply get rid of this one (and we will be able to merge that with SLOMeasurement).